mat64/dense.go

```diff
@@ -512,7 +512,13 @@ func (m *Dense) UnmarshalBinary(data []byte) error {
 	m.mat.Stride = int(cols)
 	m.capRows = int(rows)
 	m.capCols = int(cols)
-	m.mat.Data = use(m.mat.Data, m.mat.Rows*m.mat.Cols)
+
```

Comment:
I think it's OK to not limit size, but that should be noted with a warning that this should not be used from untrusted sources. (A similar warning should be placed on Dense's method.)